datapunt_api/settings.py

```diff
@@ -0,0 +1,53 @@
+from django.conf import settings
```

Comment:
Docstring with explanation what this is, and why to use it. We should consider, first implementing this in SIA --- and when proven useful move it to `drf_amsterdam`.